picoquic/sender.c

```diff
@@ -64,6 +64,7 @@ int picoquic_add_to_stream(picoquic_cnx_t* cnx, uint64_t stream_id,
             if ((stream_id & 1) != parity) {
                 ret = PICOQUIC_ERROR_INVALID_STREAM_ID;
             } else {
+#if 0
```

Comment:
Why is this #ifdef-ed out?

Reply:
That test was checking against the sending limits imposed by the peer. But that is unnecessary, and in fact it was blocking the stress test. It should be OK to just queue data on streams, and then let the transport dequeue the data when the peer opens the flow control limits.

Removing the ifdef.